data/json/recipes/recipe_weapon.json

```
@@ -2784,4 +2784,28 @@
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "leather", 30 ], [ "rope_superior_short", 2, "LIST" ] ], [ [ "scrap", 9 ], [ "razor_blade", 9 ] ] ]
   }
```

Maybe the linter's not taking a syntax error too well?
```suggestion
  },
```